src/veh_interact.h

```diff
@@ -172,6 +172,10 @@ class veh_interact
         void allocate_windows();
         void do_main_loop();
         void deallocate_windows();
+
+        /*Filter stuff*/
+        bool filter_function( const vpart_info * ) const;
```

Comment:
I wasn't sure where to put the function, though I don't think it's necessary if I don't have the complicated filter function that I previously had.